newsfragments/aip-72.significant.rst

```diff
@@ -42,6 +42,9 @@ As part of this change the following breaking changes have occurred:
 
   It is recommended that you replace such a custom operator with a deferrable sensor, a condition or another triggering mechanism.
 
+- The following operators subclassing ``SkipMixin`` have been adjusted to use the ``TaskExecution`` interface:
+
+  - ``airflow.standard.python.ShortCircuitOperator``
```

Comment:
Do users care about this? It doesn't actually change anything for them does it?

```suggestion
```

Reply:
Good call, I'll create a separate newsfragment at least for the part which is responsible for fixing short circuit in mapped tasks (https://github.com/apache/airflow/pull/44925)